storage/badger/operation/chunk_locators.go

```diff
@@ -0,0 +1,16 @@
+package operation
+
+import (
+	"github.com/dgraph-io/badger/v2"
+
+	"github.com/onflow/flow-go/model/chunks"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+func InsertChunkLocator(locator *chunks.Locator) func(*badger.Txn) error {
+	return insert(makePrefix(codeChunk, locator.ID()), locator)
+}
+
+func RetrieveChunkLocator(locatorID flow.Identifier, locator *chunks.Locator) func(*badger.Txn) error {
```

Comment:
this seems off to me, why do we need both locator and locatorID ? isn't it the second one can be computed from the first one ?

Reply:
`locator` is not an input to this function, the locator is retrieved by its id, and the _retrieved_ locator is copied into `locator`.

Reply:
yeah, the locator is not the input, it's a pointer for the function to overwrite after the chunk locator has been retrieved.